Resources/Translations/Base.lproj/Localizable.strings

```diff
@@ -274,6 +286,7 @@
 
 // App strings
 "app_name" = "Clic&Farm";
+"welcome_text" = "Welcome in Clic&Farm app! \nTo get started, sign in with your Ekylibre account…";
```

Comment:
The design asks for the "..." ?

Reply:
No, I took the same translations as android version, I have no idea what's the difference between "..." and '…'. Should we use "..." ?